odoorpc/env.py

```diff
@@ -316,6 +316,7 @@ def _create_model_class(self, model):
         fields_get = self._odoo.execute(model, 'fields_get')
         for field_name, field_data in fields_get.items():
             if field_name not in FIELDS_RESERVED:
+                field_data['context'] = self._context # pass context to new fields ...
```

Comment:
Hi, thanks for contributing.
Could you please share the use case that need to add context to field data

Reply:
Hi @flotho,

In the following sample code (some bits are redundant, they're extracted from a longer chunk of code..), using a vanilla odoorpc==0.8.0, you will get term_id = False and fiscal_pos_id = False.

Using this proposed patch, you will get the id corresponding to the requested property, depending on the requested company_id.

```
    creds['companyid'] = <sample company_id>
    rp = odoo.env['res.partner']
    rp_list = rp.with_context(allowed_company_ids=[creds['companyid']])
    move_part = rp_list.browse([<sample partner_id>])
    term_id = move_part.with_context(company=creds['companyid']).property_payment_term_id.id
    fiscal_pos_id = move_part.with_context(company=creds['companyid']).property_account_position_id.id
```

Reply:
thanks, more clear and very useful